dataland-frontend/src/App.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <section class="hero container max-w-screen-lg mx-auto pb-10 flex">
+
+    <img alt="Dataland logo" src="./assets/images/logos/dataland-logo-lr.png" class="mx-auto" >
+  </section>
+  <HelloWorld msg="Welcome to DataLand"/>
+  <APIClient/>
+
+</template>
+
+<script>
+import HelloWorld from './components/WelcomeDataland.vue'
```

Comment:
This tag should bear the same name as the component HelloWorld -> WelcomeDataland